rc_genicam_api/system.cc

```diff
@@ -370,7 +370,7 @@ std::vector<std::shared_ptr<Interface> > System::getInterfaces()
 
     // update available interfaces
 
-    if (gentl->TLUpdateInterfaceList(tl, 0, 10) != GenTL::GC_ERR_SUCCESS)
+    if (gentl->TLUpdateInterfaceList(tl, 0, GENTL_INFINITE) != GenTL::GC_ERR_SUCCESS)
```

Comment:
We can increase the timeout to e.g. 1000 ms, but I do not like to set it to infinity. If something hangs up, the application may hang forever.